src/couch_pse_tests/src/cpse_test_fold_changes.erl

```diff
@@ -170,25 +171,11 @@ do_mutation_ordering(Db, Seq, [{DocId, _OldSeq} | Rest], DocSeqAcc) ->
 
 
 shuffle(List) ->
-    random:seed(os:timestamp()),
-    Paired = [{random:uniform(), I} || I <- List],
+    Paired = [{couch_random:uniform(), I} || I <- List],
```

Comment:
`couch_rand:uniform/0` instead of `couch_random:uniform/0`?

Reply:
That is indeed a typo. Good catch. It appears that those tests don't normally run using `make eunit` -- is that correct? How do you normally run them?

Reply:
Yep, I see them now, thanks. I was trying to invoke them with the `make eunit apps=` syntax and having no luck.